loki.yaml

```diff
@@ -22,6 +22,8 @@ pipeline:
       expected-commit: 89d282c4399acc12d31bf8d0a288be7fd8e3e42d
 
   - runs: |
+      go get github.com/docker/docker@v20.10.24
```

Comment:
Can we please add the CVE-IDS as a comment , so in the next version bump it makes it easier to review if they are fixed in upstream

Reply:
Fixed!